### DISCUSSION TITLE
Making "Share as Text File" public

### DISCUSSION THREAD

**Discussion by av0c0der** [kean/Pulse#86] [Ideas]
Hey. Pulse is a great tool, thank you for making it!

With current implementation it's only possible to share the Pulse document. To be able to share logs as a text file you must present the Pulse UI which allows you to share logs as text file.

Do you think it makes sense to make that function publicly available?

**Comment by kean** [comment]
Yes, this sounds like a good idea. It should probably be part of `PulseCore`. PRs are always welcome.
